packages/mdc-theme/README.md

```diff
@@ -79,130 +71,61 @@ $mdc-theme-background: #fff;
-| `--mdc-theme-accent` | The theme accent color. |
-| `--mdc-theme-background` | The theme background color. |
-| `--mdc-theme-text-primary-on-primary` | Primary text on top of a theme primary color background. |
-| `--mdc-theme-text-secondary-on-primary` | Secondary text on top of a theme primary color background. |
-| `--mdc-theme-text-hint-on-primary` | Hint text on top of a theme primary color background. |
-| `--mdc-theme-text-disabled-on-primary` | Disabled text on top of a theme primary color background. |
-| `--mdc-theme-text-icon-on-primary` | Icons on top of a theme primary color background. |
-| `--mdc-theme-text-primary-on-accent` | Primary text on top of a theme accent color background. |
-| `--mdc-theme-text-secondary-on-accent` | Secondary text on top of a theme accent color background. |
-| `--mdc-theme-text-hint-on-accent` | Hint text on top of a theme accent color background. |
-| `--mdc-theme-text-disabled-on-accent` | Disabled text on top of a theme accent color background. |
-| `--mdc-theme-text-icon-on-accent` | Icons on top of a theme accent color background. |
-| `--mdc-theme-text-primary-on-background` | Primary text on top of the theme background. |
-| `--mdc-theme-text-secondary-on-background` | Secondary text on top of the theme background. |
-| `--mdc-theme-text-hint-on-background` | Hint text on top of the theme background. |
-| `--mdc-theme-text-disabled-on-background` | Disabled text on top of the theme background. |
-| `--mdc-theme-text-icon-on-background` | Icons on top of the theme background. |
-| `--mdc-theme-text-primary-on-light` | Primary text on top of a light-colored background. |
-| `--mdc-theme-text-secondary-on-light` | Secondary text on top of a light-colored background. |
-| `--mdc-theme-text-hint-on-light` | Hint text on top of a light-colored background. |
-| `--mdc-theme-text-disabled-on-light` | Disabled text on top of a light-colored background. |
-| `--mdc-theme-text-icon-on-light` | Icons on top of a light-colored background. |
-| `--mdc-theme-text-primary-on-dark` | Primary text on top of a dark-colored background. |
-| `--mdc-theme-text-secondary-on-dark` | Secondary text on top of a dark-colored background. |
-| `--mdc-theme-text-hint-on-dark` | Hint text on top of a dark-colored background. |
-| `--mdc-theme-text-disabled-on-dark` | Disabled text on top of a dark-colored background. |
-| `--mdc-theme-text-icon-on-dark` | Icons on top of a dark-colored background. |
+#### CSS Custom Properties
+
+CSS Custom property | Description
+--- | ---
+`--mdc-theme-primary` | The theme primary color
+`--mdc-theme-accent` | The theme accent color
+`--mdc-theme-background` | The theme background color
+`--mdc-theme-text-<TEXT_STYLE>-on-<THEME_COLOR>` | Text color for TEXT_STYLE on top of THEME_COLOR background
+`--mdc-theme-text-<TEXT_STYLE>-on-light` | Text color for TEXT_STYLE on top of light background
+`--mdc-theme-text-<TEXT_STYLE>-on-dark` | Text color for TEXT_STYLE on top of dark background
```

Comment:
Add a note:

> **NOTE:** Where <TEXT_STYLE> is directly correlated to the typographical construct the component implements. E.g.: `mdc-theme-text-hint-on-dark`

Remember our users in some cases will be new to web development and I can't tell you how long it took me to figure out what capital letters inside angle brackets meant when I was a wee lad just starting out. 😂 😭